src/content/en/updates/2019/02/trusted-types.md

```diff
@@ -0,0 +1,221 @@
+// html instanceof TrustedHTML
+document.head.innerHTML += html;
+```
+
+Here, we create a `template` policy that verifies the passed template ID
+parameter and creates the resulting HTML. The policy object `create*` functions
+return Trusted Type objects. In this case, `createHTML` returns a `TrustedHTML`
+object that the browser accepts to be used with the injection sinks that accept
+HTML.
+
+It might seem that the only improvement is in adding the following check:
+
+```js
+if (/^[0-9a-z-]$/.test(tpl)) { /* allow the tplId */ }
+```
+
+Indeed, this line is necessary to fix XSS. However, the real change is more
+profound. With Trusted Types enforcement, the *only* code that could introduce a
+DOM XSS vulnerability is the code of the policies. No other code can produce a
+value that the sink functions accept. As such, only the policies need to be
+security reviewed. In our example, it doesn't really matter where the
+`templateId` value comes from, as the policy makes sure it's correctly validated
+first - the output of this particular policy does not introduce XSS.
+
+## Limiting policies
+
+Did you notice the `*` value that we used in the `Content-Security-Policy`
+header? It indicates that the application can create arbitrary number of
+policies, provided each of them has a unique name. If applications can freely
+create a large number of policies, preventing DOM XSS in practice would be
+difficult.
+
+However, we can further limit this by specifying a whitelist of policy names
+like so:
+
+```
+Content-Security-Policy: trusted-types template
+```
+
+This assures that only a single policy with a name `template` can be created.
+That policy is then easy to identify in a source code, and can be effectively
+reviewed. With this, we can be certain that the application is free from DOM
+XSS. Nice job!
+
+In practice, modern web applications need only a small number of policies. The
+rule of thumb is to create a policy where the client-side code produces HTML or
+URLs - in script loaders, HTML templating libraries or HTML sanitizers. All the
+numerous dependencies that do not interact with the DOM, do not need the
+policies. Trusted Types assures that they can't be the cause of the XSS.
+
+## Get started {: #get-started }
+
+This is just a short overview of the API. We are working on providing more code
+examples, guides and documentation on how to migrate applications to Trusted
+Types. We feel this is the right moment for the web developer community to start
+experimenting with it.
+
+To get this new behavior on your site, you need to be
+[signed up](https://developers.chrome.com/origintrials){: .external} for the
+"Trusted Types" Origin Trial. If you just want to try it out locally, the
+experiment can be enabled on the command line:
+
+```
+    chrome --enable-blink-features=TrustedDOMTypes
+```
+
+or
+
+```
+    chrome --enable-experimental-web-platform-features
```

Comment:
remove leading spaces

Reply:
Maybe point to the UI version as well: `chrome://flags/#enable-experimental-web-platform-features`